README.md

```diff
@@ -32,26 +34,20 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 `MongoSwift` works with Swift 4.0+.
 
-### FIRST: Install the MongoDB C Driver
-Because the driver wraps the MongoDB C driver, using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. The minimum required version of the C Driver is **1.13.0**.
+### OS X and Linux
 
-On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
-`brew install mongo-c-driver`.
-
-Or on Linux, use `apt-get` to install `libmongoc` (which includes `libbson` as a dependency) and `pkg-config` (which enables Swift Package Manager to find the components):
-```
-sudo apt-get install pkg-config
-sudo apt-get install libmongoc-1.0.0
-```
+Installation on OS X and Linux is supported via [Swift Package Manager](https://swift.org/package-manager/).
 
-Alternatively, see the [installation guide](http://mongoc.org/libmongoc/current/installing.html) from libmongoc's documentation.
+#### Step 1: Install the MongoDB C Driver
+The driver wraps the MongoDB C driver, and using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. **The minimum required version of the C Driver is 1.13.0**.
 
-Next, see instructions for installation with either Swift Package Manager or CocoaPods in the following sections.
+On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
+`brew install mongo-c-driver`.
 
-### NEXT: Install the Driver Using Swift Package Manager
-*Please make sure you have followed the instructions in the previous section on installing the MongoDB C Driver before proceeding.*
+On Linux: as of now, the `libmongoc` version provided by `apt-get` is < 1.13.0 and is insufficient for use with `MongoSwift`. Please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#building-on-unix) for building and installing from source from `libmongoc`'s documentation.
```

Comment:
```suggestion
On Linux, you may be able to install both components using your package manager (e.g. `apt-get install libmongoc-1.0-0`). If the versions provided by your package manager are older than 1.13.0, please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#building-on-unix) for building and installing from source from `libmongoc`'s documentation.
```

Reply:
oops, I made my own edit and did not realize you were suggesting. now I cannot accept since the old line isn't there.
I can copy paste this in or leave it as what I wrote which is pretty similar but a bit vaguer. I don't care so let me know what you prefer.